src/lib/relayers/BiconomyRelayer.ts

```diff
@@ -0,0 +1,155 @@
+import { Biconomy } from '@biconomy/mexa';
+
+import { SupportedChainId } from '../../constants/chains';
+import {
+    BiconomyRelayerProps,
+    BiconomySendGaslessTransactionParams,
+    BiconomyWalletClientType,
+    InitBiconomyRelayerProps,
+    SendGaslessTransactionType,
+    WebHookAttributesType,
+    ZeroWalletProviderType
+} from '../../types';
+import { delay } from '../../utils/global';
+import { getTransactionReceipt } from '../../utils/provider';
+
+import { BaseRelayer } from './BaseRelayer';
+export class BiconomyRelayer implements BaseRelayer {
+    name = 'Biconomy';
+    chainId: SupportedChainId;
+    #provider: ZeroWalletProviderType;
+    #apiKey: string;
+    #fundingKey: string;
+    #biconomy = {} as any; // eslint-disable-line @typescript-eslint/no-explicit-any
+    #biconomyWalletClient?: BiconomyWalletClientType;
+    #biconomyLoading: Promise<void>;
+
+    constructor(relayerProps: BiconomyRelayerProps) {
+        this.chainId = relayerProps.chainId;
+        this.#provider = relayerProps.provider;
+        this.#apiKey = relayerProps.apiKey;
+        this.#fundingKey = relayerProps.fundingKey;
```

Comment:
What's a funding key v/s apikey?

Reply:
funding key is used to fund the gastank
api key is used to send transactions using the gastank

Reply:
It's what biconomy expose for us